src/site/_data/design/components.js

```diff
@@ -298,7 +299,12 @@ module.exports = {
     }
 
     // Passed data overwrites any context data.
+    // Regex removes whitespace and minifies: https://stackoverflow.com/a/59357436
```

Comment:
Wait—we literally have a minify step in our build. Is this important earlier than that?

Reply:
Ah, good spot because we actually don't need this now. It was playing up in a shortcode context but that won't be a problem now. I'll remove 👍